client/state/selectors/get-editor-url.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { isEnabled } from 'config';
+import { getSelectedEditor } from 'state/selectors/get-selected-editor';
+import { getSiteAdminUrl, getSiteSlug } from 'state/sites/selectors';
+import { getEditorPath } from 'state/ui/editor/selectors';
+
+export const getEditorUrl = ( state, siteId, postId = null, postType = 'post' ) => {
+	const calypsoifyGutenberg = isEnabled( 'calypsoify/gutenberg' );
```

Comment:
We probably also want to turn this off for Jetpack/VIP until we're ready to include them. 

I did something similar here for opt-in flows:
https://github.com/Automattic/wp-calypso/pull/28252/files#diff-c0750dfc4c46c43a7d5d6ffa8d810463R70

```jsx
	const siteId = getSelectedSiteId( state );
	const isVip = isVipSite( state, siteId );
	const isJetpack = isJetpackSite( state, siteId );
    const isCalypsoifyGutenbergEnabled =  isEnabled( 'calypsoify/gutenberg' ) && ! isJetpack && ! isVip;
```

Reply:
Fine to address this in a follow up PR, but let's not forget before release.

Reply:
Oh I believe this is one of those better safe than sorry cases. :)